### ISSUE TITLE
Kitty terminfo is not available as a separate package

### ISSUE THREAD

**Issue by pshirshov** [NixOS/nixpkgs#147215]
Currently terminfo data for kitty can't be installed on a headless server as a separate package.

`kitty` transitively brings in at least 11 packages with total size about 110 Mb.

Without appropriate terminfo kitty can't be used with remote Nix hosts.

Probably it should be done in Nix the same way as in Gentoo: https://gpo.zugaina.org/Search?search=kitty , they provide a separate package for kitty terminfo.

**Comment by Artturin** [comment]
It is in the terminfo output `kitty.terminfo`

**Comment by Luflosi** [comment]
While `kitty.terminfo` is available, it pulls in all of the dependencies of kitty, which are quite big because of OpenGL. It would indeed be better to have a separate terminfo package with minimal dependencies.
When I have some time, I'll see if I can do that.

**Comment by Artturin** [comment]
> While `kitty.terminfo` is available, it pulls in all of the dependencies of kitty, which are quite big because of OpenGL. It would indeed be better to have a separate terminfo package with minimal dependencies. When I have some time, I'll see if I can do that.

that has been fixed on unstable/21.11
https://github.com/NixOS/nixpkgs/issues/139756
https://github.com/NixOS/nixpkgs/pull/140051

```
nix-store --query --references $(chase result)
/nix/store/83nxpdacirjv274lihiyp18yaa3lyfsf-nix-2.4
/nix/store/9mp06ni69a44dmrjhn28mn15brdry52w-gnused-4.8
/nix/store/fvprxgcxf4px865gdjd81fbwnxcjrg41-coreutils-9.0
/nix/store/l0wlqpbsvh1pgvhcdhw7qkka3d31si7k-bash-5.1-p8
/nix/store/lhambyc1v2c7qzzr5sq7p449xs1j6pg8-gnugrep-3.7
/nix/store/q4sdqc6skbbsv0m9wmda20qc250in4b4-jq-1.6-bin
```

**Comment by Luflosi** [comment]
You're right, thanks for correcting me.
